tests/test_scope.py

```diff
@@ -179,39 +183,34 @@ def test_scope_client():
     assert scope.client == custom_client
 
 
-@pytest.mark.forked
 def test_get_current_scope():
     scope = Scope.get_current_scope()
     assert scope is not None
     assert scope.__class__ == Scope
     assert scope._type == ScopeType.CURRENT
 
 
-@pytest.mark.forked
```

Comment:
Why are we removing these forkings here?

Reply:
because it works without. And I kind of want to have fewer forked tests if possible.